buildchow/templates/versionwatch-template.yaml

```diff
@@ -0,0 +1,51 @@
+- job-template:
+    name: 'devstudio.versionwatch_{devstudio-stream}'
+    disabled: '{obj:disabled}'
+    description: |
+                 Bundle version control for JBDS<br/>
+                 Maintainers: jpeterka, nboldt<br/>
+                 Latest reports: <a href=ws/sources/vwatch/>ws/sources/vwatch/</a>*.html<br/>
+ 
+    logrotate:
+      daysToKeep: -1
+      numToKeep: 10
+      artifactDaysToKeep: -1
+      artifactNumToKeep: -1
+    properties:          
+      - github:
+          url: https://github.com/jbdevstudio/jbdevstudio-qa/
+    parameters:
+      - string:
+          name: EXCLUDE_IUS
+          default: "org.eclipse.wst.server_ui.feature"
+          description: "regex match of IUs to exclude, such as org.eclipse.wst.server_ui.feature"
+      - string:
+          name: INCLUDE_IUS
+          default: ">.*(jboss|hibernate).*"
+          description: "regex match of IUs to include, such as jboss & hibernate"
+      
+    scm:
+      - git:
+          url: git@github.com:jbdevstudio/jbdevstudio-qa.git
+          branches: 
+            - '{branch}'
+          browser: githubweb
+          browser-url: https://github.com/jbdevstudio/jbdevstudio-qa/
```

Comment:
This template is also outdated. versionwatch is now a jbosstools-* repo project.